packages/react/src/components/DashboardEditor/editorUtils.jsx

```diff
@@ -716,6 +716,7 @@ export const DashboardEditorActionsPropTypes = PropTypes.shape({
   onEditDataItem: PropTypes.func,
   dataSeriesFormActions: PropTypes.shape({
     hideAggregationsDropDown: PropTypes.func,
+    hideDataFilterDropdown: PropTypes.func,
```

Comment:
can we get an argument signature here? You may want to add some for the hideAggregationsDropdown as well. None of these functions are documented so it will be hard to write documentation for this component. I also do not see any test for this new functionality.

Reply:
@davidicus I see test for this Callback Function 
![Screen Shot 2022-09-28 at 1 52 00 PM](https://user-images.githubusercontent.com/38738726/192853918-4292dac0-a4d3-479a-a040-02aa8fa183a3.png)

Reply:
@EdmondCShaffer This test is not testing the functionality of the `hideDataFilterDropdown` function. It's just passing it in as a prop with a mocked function. 

This PR introduces a branch in the code depending on the return value of this function. We need to test that branched code. So you would mock a return value for this that would allow the branched code to render and then assert that it's there.